arkane/ess/psi4Test.py

```diff
@@ -88,6 +88,32 @@ def test_energy_from_psi4_log(self):
         log = Psi4Log(os.path.join(self.data_path, 'opt_freq_dft_ts.out'))
         self.assertAlmostEqual(log.load_energy(), -397841662.56434655, delta=1e-2)
 
+    def test_zero_point_energy_from_psi4_log(self):
+        """
+        Uses Psi4 log files to test that zero-point energies can be properly read.
+        """
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq.out'))
+        self.assertAlmostEqual(log.load_zero_point_energy(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_dft.out'))
+        self.assertAlmostEqual(log.load_zero_point_energy(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_dgt_ts.out'))
+        self.assertAlmostEqual(log.load_zero_point_energy(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_ts.out'))
+        self.assertAlmostEqual(log.load_zero_point_energy(), 228785.304, delta=1e-3)
+
+    def test_load_force_constant_matrix_from_psi4_log(self):
+        """
+        Uses Psi4 log files to test that force constant matrices can be properly read.
+        """
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq.out'))
+        self.assertAlmostEqual(log.load_force_constant_matrix(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_dft.out'))
+        self.assertAlmostEqual(log.load_force_constant_matrix(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_dgt_ts.out'))
+        self.assertAlmostEqual(log.load_force_constant_matrix(), 228785.304, delta=1e-3)
+        log = Psi4Log(os.path.join(self.data_path, 'opt_freq_ts.out'))
+        self.assertAlmostEqual(log.load_force_constant_matrix(), 228785.304, delta=1e-3)
+
```

Comment:
Thanks for adding these tests :) Can you double check the values? Both of these unit tests are currently failing ([link](https://github.com/ReactionMechanismGenerator/RMG-Py/runs/4760479782?check_suite_focus=true#step:10:6045))

Reply:
Absolutely, these are just dummies, I pushed the tentative fixes to continue the work on a different machine :)
Sorry for not commenting, I'll tag you once the PR's ready again

Reply:
Sorry for my misunderstanding!! Sounds good